src/modules/file-preview-card-group.marko

```diff
@@ -0,0 +1,772 @@
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="file-preview-card-group">
+                <ul>
+                    <li>
+                        <div class="file-preview-card">
+                            <div class="file-preview-card__body">
+                                <img
+                                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg"
+                                    alt="Image"
+                                >
+                            </div>
+                            <div class="file-preview-card__menu-action">
+                                <button
+                                    class="icon-btn"
+                                    type="button"
+                                    aria-label="Open menu"
+                                >
+                                    <svg
+                                        class="icon icon--16"
+                                        height="16"
+                                        width="16"
+                                        aria-hidden="true"
+                                    >
+                                        <use href="#icon-overflow-vertical-16"/>
+                                    </svg>
+                                </button>
+                            </div>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="file-preview-card">
+                            <div class="file-preview-card__body">
+                                <img
+                                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg"
+                                    alt="Image"
+                                >
+                            </div>
+                            <div class="file-preview-card__menu-action">
+                                <button
+                                    class="icon-btn"
+                                    type="button"
+                                    aria-label="Open menu"
+                                >
+                                    <svg
+                                        class="icon icon--16"
+                                        height="16"
+                                        width="16"
+                                        aria-hidden="true"
+                                    >
+                                        <use href="#icon-overflow-vertical-16"/>
+                                    </svg>
+                                </button>
+                            </div>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </div>
+
+    <highlight-code type="html">
+        <div class="file-preview-card-group">
+            <ul>
+                <li>
+                    <!--refer to file-preview-card-->
```

Comment:
Generally we like to include the full code, especially how its rendered above. 
It might be more verbose but its much better developer experience. 
That said, Im fine this being fixed later because its a docs change only.

Reply:
One of the ideas suggested was to solve the verbose issue as you already mentioned and also not needing to constantly update the code as changes are being made to the skin css. Yep I can fix it later in a follow up PR